pages/Blogs/index.jsx

```diff
@@ -0,0 +1,19 @@
+import Footer from "../../Components/layouts/Footer";
+import Header from "../../Components/layouts/Header";
+
+export default function Blogs() {
+  return (
+    <>
```

Comment:
here as well no need of those fragement, you can improve it by avoind this header and footer repetition in each single page